services/user/CommonSys/common/packages/wasm-loader/src/lib/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export default plugin;
+import { plugin } from './plugin.js';
+import { load } from './loader.js';
+export { plugin, load };
```

Comment:
You can do this as follows (which is the standard)
```
export { plugin } from './plugin.js';
export { load } from './loader.js';
```

Reply:
common example: https://github.com/gofractally/psibase/blob/mm/nft-sys-rs/services/user/NftSys/ui/src/components/index.ts

Reply:
Fixed